src/lib/storage.js

```diff
@@ -35,8 +35,11 @@ class Storage extends ScratchStorage {
     setProjectHost (projectHost) {
         this.projectHost = projectHost;
     }
+    setProjectToken (projectToken) {
+        this.projectToken = projectToken;
+    }
     getProjectGetConfig (projectAsset) {
-        return `${this.projectHost}/${projectAsset.assetId}`;
+        return `${this.projectHost}/${projectAsset.assetId}?token=${this.projectToken}`;
```

Comment:
Not all environments will need the token, so may not supply it. So maybe this would be better:
```suggestion
        const path = `${this.projectHost}/${projectAsset.assetId}`;
        const qs = this.projectToken ? `?token=${this.projectToken}` : '';
        return path + qs;
```